src/Microsoft.VisualStudio.ProjectSystem.Managed/Threading/Tasks/TaskDelayScheduler.cs

```diff
@@ -78,7 +78,7 @@ private async Task ThrottleAsync(Func<CancellationToken, Task> asyncFnctionToCal
                 // First we wait the delay time. If another request has been made in the interval, then this task
                 // is cancelled. To avoid unnecessary OperationCanceled exceptions it tests to see if the token has
                 // been canceled
-                await Task.Delay(TaskDelayTime);
+                await Task.Delay(TaskDelayTime, token);
```

Comment:
This method doesn't look like it expects OperationCancelledException, see comment about it.

Reply:
Quite right. I've pushed an update that addresses this. Please review again.